### ISSUE TITLE
Unable to save if created outside the app

### ISSUE THREAD

**Issue by donaldedgar** [redsolver/noteless#61]
If I create a note in Notable on my computer, synced to my phone through Syncthing, I am unable to save edits in the Noteless app. Autosave does nothing, and if I turn that feature off, the save icon is there, but unclickable and I am forced to discard any edits.

This does not happen in notes that I have created in the Noteless app. There, the save icon goes away when clicked and I am not asked to discard my edits. Nor does it happen if I create a note in Noteless, edit it in Notable, and then edit it again in Noteless. It only is a problem if created in outside the Noteless app. I even made an .md file in notepad and it was the same problem.

**Comment by redsolver** [comment]
Sorry, I could not reproduce this issue. Could you please provide the version numbers of Notable and Noteless you're using?

**Comment by donaldedgar** [comment]
I'm using Notable 1.8.4 and Noteless 1.3.1.

I just installed Notable on a new computer and the same problem occurs... Clicking the save button in Noteless doesn't register if I edited the note in Noteable. However, creating a note in the app I can edit it both ways on the desktop

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by redsolver** [comment]
I could not reproduce the issue with the same setup.
1. Could you please share the full file content of a new note created in Notable and a new note created in Noteless for comparison?
2. Please check if Syncthing sets the file attributes correctly, maybe it copies the read/write permissions from your computer and that causes Noteless to not be able to write to the file.

**Comment by donaldedgar** [comment]
It looks like it was a Syncthing problem. I had looked before but never saw it, but there's an option to ignore permissions. Once I enabled that and then recreated each file, I haven't had any problems.

Thanks for your help!

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by andreabak** [comment]
I have the same problem. Notes created in Notable (tried both 1.8 / 1.9-beta), synced using Syncthing to internal storage (external storage option in app) cannot be edited. I tried locations like `/storage/emulated/0/Notes` or `/storage/emulated/0/Android/data/net.redsolver.noteless/files`, neither work. In the second case Syncthing, with the "ignore permissions" options enabled, correctly inherited the permissions from the parent directory, so for the files the owner appeared as `a_1234` (app user) and `sdcard_rw` group.

Despite this, the "save" icon does nothing when tapped, always asks to confirm discarding changes when going back, or in "auto-save" mode just loses the changes silently.

If I disable the "external storage" option, I can edit notes normally. Maybe I can just point Syncthing to the app's `/data/data/net.redsolver.noteless/...` directory using root, but that seems like a hacky solution.

I'm on a Galaxy S10e, Android 10.

**Comment by redsolver** [comment]
I just published Version 1.4.6 (https://github.com/redsolver/noteless/releases/tag/v1.4.6) which added support for the new Android 11 folder permission system (which is already partly used in some Android 10 variants). It's possible that this new version could fix your permission issues.

**Comment by andreabak** [comment]
Yes, indeed the new version works (tested on 11). Thank you for your work 👌
